pom.xml

```diff
@@ -18,34 +18,41 @@
     <http.port>8081</http.port>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${basedir}/raml-util</ramlfiles_util_path>
-    <raml-module-builder.version>35.0.6</raml-module-builder.version>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
     <java.version>17</java.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!--Dependency Management Properties-->
-    <vertx.version>4.3.4</vertx.version>
+    <vertx.version>4.4.5</vertx.version>
     <kafkaclients.version>3.3.2</kafkaclients.version>
     <lombok.version>1.18.24</lombok.version>
     <postgres.image>postgres:12-alpine</postgres.image>
-    <log4j.version>2.19.0</log4j.version>
+    <log4j.version>2.20.0</log4j.version>
 
     <!--Dependency properties-->
     <streamx.version>0.8.1</streamx.version>
-    <rest-assured.version>5.2.0</rest-assured.version>
-    <aspectj.version>1.9.19</aspectj.version>
+    <rest-assured.version>5.3.2</rest-assured.version>
+    <aspectj.version>1.9.20.1</aspectj.version>
     <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
     <common-lang.version>2.6</common-lang.version>
     <jackson-bom.version>2.15.2</jackson-bom.version>
+    <folio-di-support.version>2.0.1</folio-di-support.version>
 
     <!--Maven plugin properties-->
-    <maven-shade-plugin.version>3.4.0</maven-shade-plugin.version>
-    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
+    <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+    <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
+    <maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
+    <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
+    <maven-shade-plugin.version>3.5.1</maven-shade-plugin.version>
+    <maven-release-plugin.version>3.0.1</maven-release-plugin.version>
     <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```

Comment:
it is being so difficult to update maven-surefire-plugin, so that I didn't change its version
![image](https://github.com/folio-org/mod-orders-storage/assets/113523904/c76f5479-58a3-4441-85c6-d4a802fabf0c)